modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,331 @@
+= Configuring Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, studio, logger, logs, log, notifications, errors, debug
+
+You can configure what gets logged, where it gets logged, and how by editing a configuration file that sits in your project.
+
+Mule logs multiple messages and specific elements in your applications' flows to help you debug and keep track of events. You can also include the xref:logger-component-reference.adoc[`<logger>` Element] anywhere in a flow and set it up to output any message you want. By creating a configuration file, you can define what kinds of messages to log, in what way (asynchronously or synchronously), and where they get logged (such as to the console, to disk, to an endpoint or to a database).
+
+For logging, Mule uses http://www.slf4j.org/[slf4j], which is a logging facade that discovers and uses a logging strategy from the classpath, such as https://logging.apache.org/log4j/2.x/[log4j2] or the JDK Logger. By default, Mule includes log4j2, which is configured with a file called `log4j2.xml`.
```

Comment:
We're trying to avoid linking out to third party sites: http://www.slf4j.org/[slf4j] because third party sites can go down, and when they do, our links and our doc builds break. A Google search for `slf4j` returns that page at the top of the search results, so the link prob isn't needed.

Reply:
Same goes for `log4j2`. You can prob replace that with "Apache Log4j 2 (log4j2)", but pls verify full name first. Then ok to use log4j2 after that.

Reply:
Probably "JDK logger"